portfolio/src/main/webapp/style.css

```diff
@@ -143,3 +143,10 @@ ol.roman-list {
   width: 200px;
 }
 
+#comments {
+  margin-left: 40%;
+  margin-right: 30%;
+  margin-top: 300px;
+  position: absolute;
+}
```

Comment:
Formatting needs some work. The images are overlapping the comments section. You can fix that in a follow up PR.

Reply:
Yeah, it's something I'll fix once I spend more time on styling! Just wanted to get the functionality working first.